packages/layerzero-v2/solana/programs/verify-contracts.md

```diff
@@ -222,9 +236,9 @@ Now, compare the program hash of the built program and the on-chain program. The
 | blocked-messagelib | 2XrYqmhBMPJgDsb4SVbjV1PnJBprurd5bzRCkHwiFCJB | 37c598b | aarch64-apple-darwin | e8f5412527e5138f626299c9b78a2e2f859d306f4c744472d7a2fde34988f3b1 |
 | dvn                | HtEYV4xB4wvsj5fgTkcfuChYpvGYzgzwvNhgDZQNh7wW | 37c598b | aarch64-apple-darwin | 98c89ebdd94b2563d3aabba118ce012965c344e98c70600f66365dae2d66de39 |
 | endpoint           | 76y77prsiCMvXMjuoZ5VRrhG5qYBrUMYTE5WgHqgjEn6 | 37c598b | aarch64-apple-darwin | 9012552d8a15d230791e2582e6320eff872a651fb110d2198020ed12e5547e74 |
-| executor           | 6doghB248px58JSSwG4qejQ46kFMW4AMj7vzJnWZHNZn | 2b168f1 | aarch64-apple-darwin | b17a413d00a54e8c666cf57797884504702ac032e8022fb0fa9c84516ef534f1 |
+| executor           | 6doghB248px58JSSwG4qejQ46kFMW4AMj7vzJnWZHNZn | 2b168f1 | aarch64-apple-darwin | d008606dd83a99a74b81cbb2ef13efccee6307a3bf2b666755533c1a95d0e1b3 |
```

Comment:
The program-hash is valid at the specific commit.

Reply:
you are correct, I realize that I did not swap the commit hashes and rebuild in between checking.  Thank you @zouguangxian